client/src/pages/AuthPage.tsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react'
-import AuthForm from '../components/Auth/AuthForm';
+import React from "react";
+import AuthForm from "../components/Auth/AuthForm";
+import { loginURL, registerURL, User } from "../config";
+import { login } from "../store/auth";
+import { useAppDispatch } from "../store/hook";
+import { useNavigate } from "react-router-dom";
+
 const AuthPage = () => {
-  return (
-    <AuthForm />
-  )
-}
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const loginHandler: (isLogin: boolean, data: User) => void = (isLogin,data) => {
+    let url: string = loginURL;
+    if (!isLogin) url = registerURL;
+    dispatch(login({ data, url }))
+      .unwrap()
```

Comment:
why the unwrap?

Reply:
i think we just need to dispatch action here and nothing else

Reply:
I've fixed this. As discussed, i'm now passing navigate as parameter to dispatch function.